game/script-events.rpy

```diff
@@ -1270,5 +1271,281 @@ label event_wintendo_twitch_game_over:
     n 1fsqajl "Well?{w=0.5}{nw}"
     extend 1fcspolesi " Get to it then,{w=0.2} [player]!"
     n 1fsqsml " Ehehe."
-    
+
+    return
+
+# Natsuki shares tips how to stay warm at chilly days
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._event_database,
+            label="event_warm_package",
+            unlocked=True,
+            conditional="jn_utils.get_total_gameplay_days() >= 21 and persistent.jn_custom_outfits_unlocked",
+            affinity_range=(jn_affinity.HAPPY, None)
+        ),
+        topic_group=TOPIC_TYPE_EVENT
+    )
+
+label event_warm_package:
+    python:
+        jn_globals.force_quit_enabled = False
+        teddy_cardigan_outfit = jn_outfits.get_outfit("jn_teddy_cardigan_outfit")
+        teddy_cardigan_outfit.unlock()
+        Natsuki.setOutfit(teddy_cardigan_outfit)
+
+    # 0 = rain, 1 = snow, 2 = default
+    $ test_int_condition = 1
+
+    if test_int_condition == 0:
+    #if jn_atmosphere.isCurrentWeatherRain() or jn_atmosphere.isCurrentWeatherThunder():
```

Comment:
Some more test code here - remember that we'd want people actually playing to have this changed based on the weather!